service/ocean/providers/aws/cluster.go

```diff
@@ -294,6 +294,7 @@ type RollSpec struct {
 	Status                       *string  `json:"status,omitempty"`
 	BatchSizePercentage          *int     `json:"batchSizePercentage,omitempty"`
 	BatchMinHealthyPercentage    *int     `json:"batchMinHealthyPercentage,omitempty"`
+	RespectPdb                   *bool    `json:"respectPdb,omitempty"`
```

Comment:
please change to:
	RespectPDB                 *bool    `json:"respectPdb,omitempty"`